discord-bot/bot.js

```diff
@@ -48,61 +51,69 @@ bot.on('message',message => {
 
 bot.on('message',message => {
 if(message.content === '!xkcd')
-{ 
+   { 
     let comicNo = Math.floor(Math.random() * (2000 - 100 + 1) + 100)
     
     axios.get(`http://xkcd.com/${comicNo}/info.0.json`).then(resp => {
     
         message.channel.send(resp.data.img)
     });
-    
-    }
+   }
     
 })
 
 //Social media commands
 
 bot.on('message',message => { 
+
    if(message.content === '!website')
    {    
     const website = new Discord.MessageEmbed().setTitle('Visit our Website').setURL('https://osdc.netlify.app/')
     message.channel.send(website)
+    return false;
```

Comment:
I don't think there's a need to return anything. Can you justify this?

Reply:
Updated

Reply:
It's still here 👀

Reply:
my bad , fixing it

Reply:
Updated !